tests/python/ColorSpaceTest.py

```diff
@@ -1,31 +1,221 @@
+        Test the setFamily() and getFamily() methods.
+        """
+
+        # Basic test
+        self.cs.setFamily("fam1")
+        self.assertEqual("fam1", self.cs.getFamily())
+
+        # Random string tests
+        for i in range(1, 10):
+            name = generate_name(10)
+            self.cs.setFamily(name)
+            self.assertEqual(name, self.cs.getFamily())
+
+    def test_equality(self):
+        """
+        Test the setEqualityGroup() and getEqualityGroup() methods.
+        """
+
+        # Basic test
+        self.cs.setEqualityGroup("match1")
+        self.assertEqual("match1", self.cs.getEqualityGroup())
+
+        # Random string tests
+        for i in range(1, 10):
+            name = generate_name(10)
+            self.cs.setEqualityGroup(name)
+            self.assertEqual(name, self.cs.getEqualityGroup())
+
+    def test_description(self):
+        """
+        Test the setDescription() and getDescription() methods.
+        """
+
+        # Basic test
+        self.cs.setDescription("this is a test")
+        self.assertEqual("this is a test", self.cs.getDescription())
+
+        # Random string tests
+        for i in range(1, 10):
+            name = generate_name(20)
+            self.cs.setDescription(name)
+            self.assertEqual(name, self.cs.getDescription())
+
+    def test_bitdepth(self):
+        """
+        Test the setBitDepth() and getBitDepth() methods.
+        """
+
+        # Known constants tests
+        bit_depths = [OCIO.Constants.BIT_DEPTH_UNKNOWN,
+                      OCIO.Constants.BIT_DEPTH_UINT8,
+                      OCIO.Constants.BIT_DEPTH_UINT10,
+                      OCIO.Constants.BIT_DEPTH_UINT12,
+                      OCIO.Constants.BIT_DEPTH_UINT14,
+                      OCIO.Constants.BIT_DEPTH_UINT16,
+                      OCIO.Constants.BIT_DEPTH_UINT32,
+                      OCIO.Constants.BIT_DEPTH_F16,
+                      OCIO.Constants.BIT_DEPTH_F32]
+
+        for bit_depth in bit_depths:
+            self.cs.setBitDepth(bit_depth)
+            self.assertEqual(bit_depth, self.cs.getBitDepth())
+
+        # Random string tests
+        for i in range(0, 10):
+            name = generate_name(10)
+            self.cs.setBitDepth(name)
+            self.assertEqual('unknown', self.cs.getBitDepth())
+
+        # Wrong type tests
+        with self.assertRaises(TypeError):
+            self.cs.setBitDepth(None)
+
+    def test_data(self):
+        """
+        Test the setIsData() and getIsData() methods.
+        """
+
+        # Boolean tests
+        is_datas = [True, False]
+        for is_data in is_datas:
+            self.cs.setIsData(is_data)
+            self.assertEqual(is_data, self.cs.isData())
+
+        # Wrong type tests
+        wrong_is_datas = [['test'],
+                          'test']
+        for wrong_is_data in wrong_is_datas:
+            with self.assertRaises(TypeError):
+                self.cs.setIsData(wrong_is_data)
```

Comment:
I'm assuming this is failing in CI because most types can evaluate to bool, which this method is expecting.